mdd/python/translation/openconfig_xe/xe_interfaces.py

```diff
@@ -138,7 +138,7 @@ def xe_process_interfaces(self) -> None:
                             physical_interface.switchport.delete()
                         xe_interface_aggregation(self, interface, port_channel)
         # Physical and Sub-interfaces
-        elif interface.config.type == 'ianaift:ethernetCsmacd' and interface.subinterfaces.subinterface:
```

Comment:
This is the default for ethernetCsmacd and should be kept under more specific ethernet matches.